no-computer.md

```diff
@@ -1,55 +1,68 @@
+
+The DSL is massively extensible, which is both a blessing and a curse. The field itself can be broken up into (arbitrarily small) zones and then, if a physics engine is applied, we could actually simulate where a ball would go, with what exit velocity, and so on.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+I think at first, the program could look up how hitters have dealt with certain pitches in certain zones in the past, and also see how well pitchers have successfully located pitches in the desired location in the past (baseball has lots and lots of data -- this would be easy to look up/reference), in order to provide some sort of probability of various outcomes. Eventually a physics simulation could be added, which would actually look at contact with the bat and provide a prediction for where a ball would land.
+
+Errors would possibly include insufficient data (at which point perhaps the program could harbor a bad guess or something like that), or if the user inputs unreasonable inputs, which the program wouldn't be able to do anything with, such as the pitcher throwing 200MPH or something like that. I suppose the program would simply pop up an alert/error
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to make predictions as to how batters will do against certain pitchers (as this is the intended functionality). It should be difficult to ask for unrealistic projections, such as impossibly fast pitchers. It should be impossible to create non-standard baseball outcomes, like a ball hitting a bird or having fan interference disrupt play.
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
-
+There is [baseball-sim](https://www.sabersim.com/mlb), which is very similar to what I described! (I didn't know about it before I started writing). While the writer does pretty much everything I described above, he still feels the simulator is not very useful -- he writes about his thoughts [here.](http://www.hardballtimes.com/10-lessons-i-learned-from-creating-a-baseball-simulator/)
```

Comment:
Look at Strat-O-Matic! (_I'm starting to feel like a salesman for them... where's my commission?!_)